modules/ROOT/pages/latest-overview-concept.adoc

```diff
@@ -1,68 +1,201 @@
-= Reviewing API Manager Concepts
+= API Manager Overview
```

Comment:
Thanks so much for the awesome editorial review, Kim! I implemented most of your feedback. The only item I didn't incorporate was adding more context to xref links. I was thinking that in most cases this would create redundancy, as the surrounding topics provide information on what can be found in the links.